proposals/nnnn-apt-clarification.md

```diff
@@ -0,0 +1,146 @@
+         <description>Describes different sampling options for PerformAudioPassThru.</description>
+-        <element name="8KHZ" internal_name="SamplingRate_8KHZ"/>
+-        <element name="16KHZ" internal_name="SamplingRate_16KHZ"/>
+-        <element name="22KHZ" internal_name="SamplingRate_22KHZ"/>
+-        <element name="44KHZ" internal_name="SamplingRate_44KHZ"/>
++        <element name="8KHZ" internal_name="SamplingRate_8KHZ">
++            <description>Sampling rate of 8000 Hz.</description>
++        </element>
++        <element name="16KHZ" internal_name="SamplingRate_16KHZ">
++            <description>Sampling rate of 16000 Hz.</description>
++        </element>
++        <element name="22KHZ" internal_name="SamplingRate_22KHZ">
++            <description>Sampling rate of 22050 Hz.</description>
++        </element>
++        <element name="44KHZ" internal_name="SamplingRate_44KHZ">
++            <description>Sampling rate of 44100 Hz.</description>
++        </element>
+     </enum>
+ 
+     :
+ 
+     <enum name="BitsPerSample">
+         <description>Describes different quality options for PerformAudioPassThru.</description>
+-        <element name="8_BIT" internal_name="BitsPerSample_8_BIT"/>
+-        <element name="16_BIT" internal_name="BitsPerSample_16_BIT"/>
++        <element name="8_BIT" internal_name="BitsPerSample_8_BIT">
++            <description>Audio sample is 8 bits wide, unsigned.</description>
++        </element>
++        <element name="16_BIT" internal_name="BitsPerSample_16_BIT">
++            <description>Audio sample is 16 bits wide, signed, and in little endian.</description>
++        </element>
+     </enum>
+ 
+     :
+ 
+     <enum name="AudioType">
+         <description>Describes different audio type options for PerformAudioPassThru.</description>
+-        <element name="PCM" />
++        <element name="PCM">
++            <description>Linear PCM.</description>
++        </element>
+     </enum>
+ 
+     :
+ 
+    <struct name="AudioPassThruCapabilities">
+         <description>
+             Describes different audio type configurations for PerformAudioPassThru.
+             e.g. {8kHz,8-bit,PCM}
++            The audio is recorded in monaural.
+         </description>
+        <param name="samplingRate" type="SamplingRate" mandatory="true"/>
+        <param name="bitsPerSample" type="BitsPerSample" mandatory="true"/>
+        <param name="audioType" type="AudioType" mandatory="true"/>
+    </struct>
+```
+
+### Modification of Core
+
+- Update `AudioStreamSenderThread` class to skip RIFF header when it reads the audio data (either recorded or prepared) from file.
+- Update the GStreamer pipeline in `FromMicToFileRecorderThread` class to make sure that it always outputs audio in monaural.
+
+
+## Potential downsides
+
+The author does not come up with any potential downsides.
+
+
+## Impact on existing code
+
+- The changes in interface files do not impact on any implementation.

```

Comment:
Please update to "The changes in interface files do not have an impact on any implementation."